packages/kbn-test/src/es/test_es_cluster.ts

```diff
@@ -139,3 +148,28 @@ export function createTestEsCluster(options: TestClusterFactoryOptions) {
     }
   })();
 }
+
+/**
+ * Takes in two arrays of 'key=val' strings, allowing values from the second array
+ * to override values in the first.
+ *
+ * @example
+ *
+ * mergeDefaultArgs(['foo=a', 'bar=b'], ['foo=c', 'baz=d']); // returns ['foo=c', 'bar=b', 'baz=d']
+ */
+function mergeDefaultArgs(defaultArgs: string[], customArgs: string[]): string[] {
```

Comment:
Needed a way to ensure all `customArgs` could override the defaults we set above.